detox/test/package.json

```diff
@@ -30,14 +30,14 @@
     "verify-artifacts:android": "jest ./scripts/verify_artifacts_are_not_missing.android.test.js --testEnvironment node"
   },
   "dependencies": {
-    "@react-native-community/async-storage": "^1.12.0",
+    "@react-native-async-storage/async-storage": "^1.17.3",
```

Comment:
The package name here has changed - see https://www.npmjs.com/package/@react-native-community/async-storage